Modules/ThirdParty/MetaIO/src/MetaIO/src/metaCommand.cxx

```diff
@@ -1722,8 +1722,8 @@ bool MetaCommand::Parse(int argc, char* argv[])
   std::string tag = "";
   std::string args;
 
-  unsigned int currentField = 0; // current field position
-  int currentOption = 0; // id of the option to fill
+  unsigned long currentField = 0; // current field position
```

Comment:
May be _long long_, otherwise there is no effect on Windows.

Reply:
I changed `currentOption` to long because `GetOptionId` returns `long`.